src/middleware/error/handleError.js

```diff
@@ -1,11 +1,11 @@
+// eslint-disable-next-line import/no-unresolved
 import * as Sentry from '@sentry/node';
 import Error from './ErrorHandler';
 import logger from '../../logger';
-
 // eslint-disable-next-line no-unused-vars
 function handleError(err, req, res, next) {
   if (err instanceof Error) {
-    res.status(err.statusCode).json(err.message);
+    res.status(err.statusCode).json({ message: err.message });
```

Comment:
why do we need to assign it as message here ?

Reply:
I guess this can be removed, I put there when figuring out reasons why it couldn't show the error messages in tests

Reply:
yes pls remove this line because frontend codes uses the original format when receiving error message from server